zipline/test_algorithms.py

```diff
@@ -485,6 +485,11 @@ def initialize(self, max_leverage=None):
         self.set_max_leverage(max_leverage=max_leverage)
 
 
+class SetMinLeverageAlgorithm(TradingAlgorithm):
+    def initialize(self, min_leverage=None, timedelta=None):
```

Comment:
Should these just be required if passing them as None will isn't actually supported?